packages/client/src/Routes/AppRoutes.ts

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
-import { Home, Login, Register, NotFound, Unauthorized } from '../pages';
+import {
+  Home,
+  Login,
+  Register,
+  NotFound,
+  Unauthorized,
+  Registered,
+} from '../pages';
 
 export enum AppRoute {
   Home = '/',
   Login = 'login',
   Register = 'register',
+  Registered = 'registered',
```

Comment:
what does this app route do? upon successful registration, from registered page, will the user be redirected to Home page automatically ?

Reply:
Nothing too fancy, just when registration is successful it goes to a "thanks for registering" page, thinking of either having some text on there and a link to login, or maybe just the login form